scripts/components/Sidebar.js

```diff
@@ -29,9 +29,10 @@ export default class Sidebar extends Component {
                 "list-group-item",
                 params.name === name ? "active" : "",
               ].join(" ");
+              const label = collections[name].name || name;
               return (
                 <Link key={i} to={`/collections/${name}`} className={classes}>
-                  {name + (!collections[name].synced ? "*" : "")}
+                  {label + (!collections[name].synced ? "*" : "")}
```

Comment:
Well `addons` will now be listed in the sidebar instead of "add-ons" if that's your question :) So to summarize:

- The collection key is the technical name (the Kinto collection name used in the request URLs);
- The name propery is the *display name* for that collection in the menu.